quinn-proto/src/connection/mod.rs

```diff
@@ -728,6 +761,9 @@ where
     /// Executes protocol logic, potentially preparing signals (including application `Event`s,
     /// `EndpointEvent`s and outgoing datagrams) that should be extracted through the relevant
     /// methods.
+    ///
+    /// It is most efficient to call this only for each `Instant` output by `poll_timeout`;
+    /// however spurious extra calls will simply no-op and therefore are safe.
```

Comment:
Note: I should reword this slightly so that you can't misinterpret it to do something like `handle_timeout(poll_timeout())` which would be wrong.

Reply:
Maybe s/for each `Instant`/after passing `Instant`s/?